devstack/plugin.sh

```diff
@@ -11,7 +11,7 @@ function gbp_configure_heat {
 }
 
 function gbp_configure_neutron {
-    iniset $NEUTRON_CONF group_policy policy_drivers "implicit_policy,resource_mapping,chain_mapping"
+    iniset $NEUTRON_CONF group_policy policy_drivers "implicit_policy,resource_mapping"
     iniset $NEUTRON_CONF group_policy extension_drivers "proxy_group"
```

Comment:
In gbp master branch, we need chain_mapping also. We should not change anything in this function "gbp_configure_neutron"